docs/pages/clients/registry.md

```diff
@@ -301,6 +298,18 @@ rows = client.search()
 # [['vanessa/tacos', 'latest', 'Dec 28, 2017 02:56AM']]
 ```
 
+## Delete
+When authorized to do so (see [Push](#push), it is possible to delete images from the registry. Don't forget to confirm with `y`, or pass `--force` to avoid being prompted:
+
+```python
+$ sregistry delete vanessa/tacolicious:gobacktosleep
```

Comment:
Looks like this should be bash too (I take responsibility for this error, since you fixed it above!)

Reply:
Fixed